src/Uno.UI.RuntimeTests/Helpers/ImageAssert.cs

```diff
@@ -0,0 +1,334 @@
+﻿#nullable enable
```

Comment:
Could we just link the file instead of copying it ? (Abstracting the `ScreenshotInfo`)

Reply:
Linking does not work, there are too many differences, but more splitting could make more things reuseable.